packages/webapp/src/components/Form/Input/index.jsx

```diff
@@ -106,6 +114,7 @@ const Input = ({
       {unit && <div className={styles.unit}>{unit}</div>}
       {currency && <div className={styles.currency}>{currency}</div>}
       <input
+        onFocus={inputType === 'date' ? showPickerHandler() : null}
```

Comment:
We don't always want a date input to be focused, so I think this component should take `autoFocus`(I used what you passed from `TaskDate` component) as prop.

I don't think those date inputs in the screenshot below need focus on them, and we should be able to focus on one of them when we want to.
![Screenshot 2023-02-02 at 11 40 28 AM](https://user-images.githubusercontent.com/33141219/216432978-9e15dcfe-dbf0-4297-a0f7-f317f04af3a4.png)